Makefile

```diff
@@ -344,7 +344,7 @@ setup-local-debugging:
 auth:
 	$(call target_title,"Setting up Azure Active Directory") \
 	&& . ${MAKEFILE_DIR}/devops/scripts/check_dependencies.sh nodocker,env \
-	&& . ${MAKEFILE_DIR}/devops/scripts/create_aad_assets.sh
+	&& ${MAKEFILE_DIR}/devops/scripts/create_aad_assets.sh
```

Comment:
Out of curiosity :)  why did you remove the dot here? I understand you don't need the output to be on the current context, but in the future we might add more commands here... So was just wondering what affect did it have?

Reply:
We have two types of script - those that we want to invoke and those that we want to `source` (the `.` form is shorthand).

When we use `source`, we run the contents of the script in the original shell context. This is useful when we want to set environment variables from files or when we want to load shared function definitions. 

The rest of the time (such as here), we just want to invoke the script which makes the control flow more natural. For example, an `exit` in a script called normally will exit that script and return to the caller. In a `source`d script, the `exit` would exit from the _calling_ script because the script is run in the caller's context.